Add building alternative version with page numbers

qwrtln opened the pull request:
This PR adds building an alternative version with page numbers. See the proposed readme file here: https://github.com/Heegu-sama/Homm3BG/blob/c716c55b7d16c271a0aec6a79f47357f4fcee712/README.md

Image snapshot below:
![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/ad20ebce-2370-4ca8-9795-127d425cca91)

csagataj2 reviewed (approved):
Extra emojis are always welcome :D

qwrtln commented:
> Extra emojis are always welcome :D

Just as you write, I fixed some errors and introduced even more emojis ;-)

![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/52720d38-8596-4d23-931c-b91e3b89bf28)
